WebApplicationDemo/Startup.cs

```diff
@@ -70,6 +71,12 @@ public override void ConfigureServices(IServiceCollection services)
 			})
 			;
 
+			//HealthChecks
+			services.AddArkHealthChecks()
```

Comment:
I think AddArkHealthChecks() should be done in the common Startups.
The application should only Add more checks, but UI and Endpoint should be there by default.